scripts/get_native_libraries.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import urllib.request
+import shutil
+import zipfile
+
+from os import path, remove
+
+
+def download_native_libraries(name: str, version: str, destination: str):
+    header = f'{destination}/{name}.h'
+    mac = f'{destination}/libcosmian_{name}.dylib'
+    linux = f'{destination}/libcosmian_{name}.so'
+    windows = f'{destination}/cosmian_{name}.dll'
+    jni_libs = 'android/src/main/jniLibs'
+    armeabi_v7a = f'{jni_libs}/armeabi-v7a/libcosmian_{name}.so'
+    arm64_v8a = f'{jni_libs}/arm64-v8a/libcosmian_{name}.so'
+    x86 = f'{jni_libs}/x86/libcosmian_{name}.so'
+    x86_64 = f'{jni_libs}/x86_64/libcosmian_{name}.so'
+    ios = f'ios/libcosmian_{name}.a'
```

Comment:
Could you use a dict to store the `source => destination` so you can loop below instead of copy/paste `shutil.copyfile` multiple times?